kbase-extension/static/kbase/js/widgets/narrative_core/upload/fileUploadWidget.js

```diff
@@ -85,20 +85,23 @@ define([
                     $dropzoneElem.find('#global-info').css({'display': 'inline'});
                     $dropzoneElem.find('#upload-message').text(this.makeUploadMessage());
 
+                    // If there is a button already in the area, it has to be removed,
+                    // and appened to the new document when additional errored files are added.
+                    if ($dropzoneElem.find('#clear-all-btn').length){
+                        this.deleteClearAllButton();
+                        $dropzoneElem.append(this.makeClearAllButton());
+                    }
+
                 })
                 .on('success', (file, serverResponse) => {
+                    var $successElem = $(file.previewElement);
+                    $successElem.find('#upload_progress_and_cancel').hide();
+                    $successElem.find('#dz_file_row_1').css({'display': 'flex', 'align-items': 'center'});
+                    $successElem.find('#success_icon').css('display', 'flex');
+                    $successElem.find('#success_message').css('display', 'inline');
```

Comment:
Can we change these so that rather than adding/removing inline styles, we are adding/removing a class instead? That way, all the css info can go in the stylesheet.

Will there only ever be one file loading at a time -- i.e. will the IDs always be unique?
e.g.

Reply:
So this code only changes the file template (that is shown in dropzone) for a the single file. The default template, which creates each file's html won't be changed by the .on('errorr'... changes. I don't know if that made sense. If it didn't, let me know and I can explain on a quick zoom call. I can work on making the styling changes in classes instead of jquery stuff :)